intelmq/bots/parsers/netlab_360/parser_magnitude.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""
+See docs/Feeds.md
+"""
+
+import sys
+
+from datetime import datetime
+from intelmq.lib.bot import ParserBot
+from intelmq.lib.harmonization import FQDN, IPAddress, URL
+from intelmq.lib.message import Event
+
+
+class Netlab360MagnitudeParserBot(ParserBot):
+
+    def parse_line(self, line, report):
+        if line.startswith('#') or len(line) == 0:
+            self.tempdata.append(line)
+        else:
+            lvalue = line.split('\t')
+            event = Event(report)
+
+            event.add('classification.identifier', lvalue[0].lower())
+            event.add('time.source',
+                      datetime.utcfromtimestamp(int(lvalue[1])).strftime('%Y-%m-%dT%H:%M:%S+00:00'))
```

Comment:
I'll look into this more, before making the change to what it is now I kept getting an error in the tests about being out of range or something.